### PR TITLE
Added 100 more sentences

### PR DESCRIPTION
**Pull request by reinhart1010**
As with previous pull requests, these new texts are made to add new words and popularize rare-occuring words in current sample sentences database. As of now, about 62% of these words occurs only once.

My Common Voice statistics is now moved to [reinhart1010/common-text](https://github.com/reinhart1010/common-text). The old [gist](https://gist.github.com/reinhart1010/8717b4d765ee4e5eb0765aa4c9955016) will not be updated anymore.

In addition, I could write sample sentences in Indonesian (Bahasa Indonesia), in case that the Common Voice needs it.

### CODE REVIEW

**Comment by reinhart1010** [comment]
**Update**: 60 new sentences are being added into this PR.

File: server/data/reinhart1010.txt [hunk 1]
```diff
@@ -48,3 +48,63 @@ She accuse her friend for cheating in class.
...
+The element antimony is mainly used in flame retardants.
+Parrafin wax is an important ingredient of a candle.
+Are you still using your computer for the research?
+I’ve got an announcement to make.
+What is it, boss?
+From now on, your salaries is going to be lowered by 20 percent.
+You can’t do that, it is written in our contract.
+Security, kick this guy out!
+This is unacceptable, we must do some thing!
+"We haven’t reach our target yet", said the boss.
```
Comment (samlh) [inline server/data/reinhart1010.txt]: reach -> reached

File: server/data/reinhart1010.txt [hunk 2]
```diff
@@ -48,3 +48,63 @@ She accuse her friend for cheating in class.
...
+There are nineteen students in this Economics class.
+The Central-Midlevels escalator system in Hong Kong consists of twenty escalators.
+I suspect that Irene erased my backup copy of my report.
+My niece sent me a nice photo of Moscow via email.
+The element antimony is mainly used in flame retardants.
+Parrafin wax is an important ingredient of a candle.
+Are you still using your computer for the research?
+I’ve got an announcement to make.
+What is it, boss?
+From now on, your salaries is going to be lowered by 20 percent.
```
Comment (samlh) [inline server/data/reinhart1010.txt]: is -> are

File: server/data/reinhart1010.txt [hunk 3]
```diff
@@ -48,3 +48,63 @@ She accuse her friend for cheating in class.
...
+You can’t do that, it is written in our contract.
+Security, kick this guy out!
+This is unacceptable, we must do some thing!
```
Comment (samlh) [inline server/data/reinhart1010.txt]: some thing -> something

File: server/data/reinhart1010.txt [hunk 4]
```diff
@@ -48,3 +48,63 @@ She accuse her friend for cheating in class.
...
+"We haven’t reach our target yet", said the boss.
+We are doing an overtime ban as we are dissatisfied with the boss' decisions.
+How bad can a boss be?
+How many times should I tell you, don’t play Space War during work!
+Have you finish with the Annual Report, Jane?
```
Comment (samlh) [inline server/data/reinhart1010.txt]: finish -> finished

File: server/data/reinhart1010.txt [hunk 5]
```diff
@@ -48,3 +48,63 @@ She accuse her friend for cheating in class.
...
+That’s what you get for testing my patience.
+I have a list of candidate to replace you.
```
Comment (samlh) [inline server/data/reinhart1010.txt]: candidate -> candidates

File: server/data/reinhart1010.txt [hunk 6]
```diff
@@ -48,3 +48,63 @@ She accuse her friend for cheating in class.
...
+I just opened the link you sent to me.
+The on-line dictionary is much easier and faster than paper dictionary.
```
Comment (samlh) [inline server/data/reinhart1010.txt]: than -> than a OR than the

File: server/data/reinhart1010.txt [hunk 7]
```diff
@@ -48,3 +48,63 @@ She accuse her friend for cheating in class.
...
+Check the main features on the website.
+View the reviews, comments and ratings on the website.
+Check the website from viruses.
```
Comment (samlh) [inline server/data/reinhart1010.txt]: from -> for OR doesn't have

File: server/data/reinhart1010.txt [hunk 8]
```diff
@@ -48,3 +48,63 @@ She accuse her friend for cheating in class.
...
+I was told by my instructor to use a nichrome wire to burn metal solids in a Bunsen burner.
+The candles brightened the dark room near to the garage.
+These home-made sculptures seems interecting for me.
+Eleven scorpions were collected by a hunter.
+At least twelve scripts were found in this chest.
+Thirteen totems were spread across this island.
+She brought fourteen rubber ducks to play in the swimming pool.
+My office is located on level fifteen.
+Sixteen participants are involved in a chess tournament.
+Seventeen gardeners are working in this afternoon.
```
Comment (samlh) [inline server/data/reinhart1010.txt]: working in this -> working this

File: server/data/reinhart1010.txt [hunk 9]
```diff
@@ -48,3 +48,63 @@ She accuse her friend for cheating in class.
...
+I had some problems in activating this coupon.
+That is the weirdest dress I have found in this store.
+She is absent from working in the drug store today.
+She stored a collection of dictionaries and encyclopedia books in this cupboard.
+He have modified a t-rex game for a Christmas event.
+My nephew works as a barista in a small cafe in Edinburgh.
+I really love to watch live streams of developer conferences.
+I was told by my instructor to use a nichrome wire to burn metal solids in a Bunsen burner.
+The candles brightened the dark room near to the garage.
+These home-made sculptures seems interecting for me.
```
Comment (samlh) [inline server/data/reinhart1010.txt]: seems interecting for me -> seem interesting to me

File: server/data/reinhart1010.txt [hunk 10]
```diff
@@ -48,3 +48,63 @@ She accuse her friend for cheating in class.
...
+She decided to study Accounting at a university in Portugal.
+Elena took a photo of the fountain in front of the Eiffel Tower.
+Nara took a flight from South Africa to Japan.
+I saw too many advertisements covering this website.
+I think I saw some actors were gathering at a large restaurant.
+I had some problems in activating this coupon.
+That is the weirdest dress I have found in this store.
+She is absent from working in the drug store today.
+She stored a collection of dictionaries and encyclopedia books in this cupboard.
+He have modified a t-rex game for a Christmas event.
```
Comment (samlh) [inline server/data/reinhart1010.txt]: He have -> He has

File: server/data/reinhart1010.txt [hunk 11]
```diff
@@ -48,3 +48,63 @@ She accuse her friend for cheating in class.
...
+The on-line dictionary is much easier and faster than paper dictionary.
+Check the main features on the website.
+View the reviews, comments and ratings on the website.
+Check the website from viruses.
+Check whether the website is funded by ads, donations, or others.
+Check the person and/or company who owns this website. 
+Find how useful the information is in the website.
+Verify the contents of the website.
+Check the copyright, trademark and patent information contained on the website.
+Be careful in handling all the chemicals which are corrosive and irritant.
```
Comment (samlh) [inline server/data/reinhart1010.txt]: corrosive and irritant -> corrosive or irritant

**Comment by reinhart1010** [comment]
@samlh Thanks for your suggestions. I will reconsider these changes (and add more sentences) by this week.

**Comment by mikehenrty** [comment]
thanks @reinhart1010 !

**Pull request merged**